Lib/http/server.py

```diff
@@ -692,10 +704,11 @@ class SimpleHTTPRequestHandler(BaseHTTPRequestHandler):
         '.xz': 'application/x-xz',
     }
 
-    def __init__(self, *args, directory=None, **kwargs):
+    def __init__(self, *args, directory=None, response_headers=None, **kwargs):
         if directory is None:
             directory = os.getcwd()
         self.directory = os.fspath(directory)
+        self.response_headers = response_headers
```

Comment:
Clarify as an internal private attribute:

```suggestion
        self._response_headers = response_headers
```

Or document _SimpleHTTPRequestHandler.response_headers_ as a public attribute.

Reply:
So far I have intended `response_headers` to be a public attribute. Do you mean add documentation to the docstring of `SimpleHTTPRequestHandler` or more documentation in `Doc/library/http.server.rst`?